### ISSUE TITLE
ADXL372_SCALEG

### ISSUE THREAD

**Issue by goamberg** [ezra-varady/adxl372#1]
I have a question to ADXL372_SCALEG = 0.09768

I get how the calculation has been done. But in the adxl372 data sheet it says "12-bit output at 100 mg/LSB scale factor".

Wouldnt SCALEG then just be a simple 0,1 factor and from there times 9,81 to get m/s². 
Or is it something I just do not have on my radar?

I know it doesnt make a difference, but I am learning ;-) 
Anyways, your work here is awesome and helped a lot with tinkering with this Sensor
Cheers
Goam

**Comment by ezra-varady** [comment]
Yeah this seems right. I'm not sure what I was thinking. Feel free to submit a PR, otherwise I'll change it when I get a minute
